android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/MyPoliciesActivity.java

```diff
@@ -0,0 +1,74 @@
+package com.liferay.mobile.formsscreenletdemo.view;
+
+import android.content.Intent;
+import android.graphics.Color;
+import android.support.design.widget.Snackbar;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+
+import com.liferay.mobile.formsscreenletdemo.R;
+import com.liferay.mobile.screens.asset.AssetEntry;
+import com.liferay.mobile.screens.asset.list.AssetListScreenlet;
+import com.liferay.mobile.screens.base.list.BaseListListener;
+import com.liferay.mobile.screens.util.AndroidUtil;
+
+import java.util.List;
+
+/**
+ * @author Vitória Mendes
+ * @author Luísa Lima
+ */
+public class MyPoliciesActivity extends AppCompatActivity implements BaseListListener<AssetEntry> {
+
+    private AssetListScreenlet assetListScreenlet;
+    private Toolbar toolbar;
+    private final String ENTRY_ID = "entryId";
```

Comment:
Move this variable to a file with Constants used only for Keys or use ENTRY_ID_KEY

Reply:
Okay